index.html

```diff
@@ -122,32 +122,32 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
-        <!-- ________ *Peter* Contributor card START ________  -->
+        <!-- ________ *Furkan* Contributor card START ________  -->
         <div class="card">
-          <p class="name">Chard Peter</p>
+          <p class="name">Furkan Murat</p>
           <p class="contact">
             <i class="fab fa-x-twitter"></i>
-            <a href="https://x.com/chardpeter1" target="_blank">@ChardPeter1</a>
+            <a href="https://twitter.com/Furkannnmurat" target="_blank">Furkannnmurat</a>
             <i class="fab fa-github"></i>
-            <a href="https://www.github.com/chardpeter" target="_blank">@chardpeter</a>
+            <a href="https://www.github.com" target="_blank">Your handle</a>
```

Comment:
It seems you've forgotten to put your handle here - replace the link with a link to your specific github account, and put your github account in place of "Your handle"